plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/toolwindow/AmazonQToolWindow.kt

```diff
@@ -25,8 +24,7 @@ class AmazonQToolWindow private constructor(
     val component
         get() = chatPanel.component
 
-    fun disposeAndRecreate() {
-        Disposer.dispose(chatPanel)
+    fun recreatePanel() {
         chatPanel = AmazonQPanel(project, scope)
```

Comment:
@manodnyab will it help if we assign the new panel first then dispose the stale one?

Reply:
maybe not, just tested

Reply:
#5729 would fix the crash issue

Reply:
Removing this part, if 5729 fixes the crash